scripts/management/MgmtDB.py

```diff
@@ -117,6 +130,34 @@ def update_entries_live(
 
         return True
 
+    @staticmethod
+    def find_dependant_task(self, cur, entry):
```

Comment:
A static method with self?

Reply:
That's the last time I listen to IDE feedback... (fixed lol)